### ISSUE TITLE
Clean theme Iframes overflowing article container

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4304]
This has been split off from #4261 as it has similar symptoms but not from the same cause.

> Not just References - other content overflows too.  This was noted while testing another fix on my dev install, so this article does not exist on Clean, it was the one flagged up for a different problem in #3911 then downloaded for use as a test subject, but it illustrates the problem.
> 
> 
> 
> <img width="922" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/0675a606-038f-4902-b554-988f8333b287">
> 
>  

 _Originally posted by @StephDriver in [#4261](https://github.com/BirkbeckCTP/janeway/issues/4261#issuecomment-2191816366)_

- [ ] content inside the article should be constrained by the article container, and not overflow into the sidebar.

**Comment by StephDriver** [comment]
This was discussed at backlog refinement today, and was split from #4261.

**Comment by StephDriver** [comment]
branch has been merged into master
